resources/definitions/jellybox.def.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "jellybox",
+    "version": 2,
+    "name": "JellyBOX",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "IMADE3D",
+        "manufacturer": "IMADE3D",
+        "category": "Other",
+        "platform": "jellybox_platform.stl",
+        "platform_offset": [ 0, 0, 0],
```

Comment:
Currently the platform seems to be _just_ a little bit too high, as it goes into the build volume and makes the build plate appear foggy. It would help if you made the second coordinate slightly negative, moving the build plate down. `[0, -0.3, 0]` seems to work well.